code/datums/traits/good.dm

```diff
@@ -194,3 +194,90 @@
 	if(!D) //if their current mob doesn't have a bank account, likely due to them being a special role (ie nuke op)
 		return
 	D.welfare = TRUE
+
+/datum/quirk/prepared	//thanks to the coder of "Family Heirloom!"
+	name = "Came Prepared"
+	desc = "You knew what job you were getting at the station, and brought something helpful to your new job!"
+	value = 4
+	var/obj/item/prepared
+	var/where
+
+/datum/quirk/prepared/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/obj/item/prepared_job
+	switch(quirk_holder.mind.assigned_role)
+		//Service jobs
+		if("Clown")
+			prepared_job = pick(/obj/item/reagent_containers/spray/waterflower/lube, /obj/item/reagent_containers/glass/bottle/fake_gbs, /obj/item/disk/nuclear/fake, /obj/item/card/emagfake/honkmag)
+		if("Mime")
+			prepared_job = pick(/obj/item/reagent_containers/food/snacks/baguette, /obj/item/card/emagfake/honkmag, /obj/item/disk/nuclear/fake)
+		if("Janitor")
+			prepared_job = pick(/obj/item/storage/bag/trash, /obj/item/reagent_containers/spray/cleaner, /obj/item/janiupgrade)
+		if("Cook")
+			prepared_job = pick(/obj/item/kitchen/knife/butcher, /obj/item/kitchen/rollingpin,/obj/item/reagent_containers/food/snacks/dough)
+		if("Botanist")
+			prepared_job = pick(/obj/item/multitool, /obj/item/reagent_containers/glass/bottle/mutagen, /obj/item/seeds/random, /obj/item/circuitboard/machine/chem_dispenser)
+		if("Bartender")
+			prepared_job = pick(/obj/item/storage/box/stunslug, /obj/item/storage/box/rubbershot, subtypesof(/obj/item/reagent_containers/food/drinks))
+		if("Curator")
+			prepared_job = pick(/obj/item/stack/sheet/mineral/wood, /obj/item/pen/fourcolor, /obj/item/pen/invisible)
+		if("Assistant")
+			prepared_job = pick(/obj/item/stack/tile/fakepit, /obj/item/storage/box/fakesyndiesuit, /obj/item/stack/tile/fakespace)
+		//Security/Command
+		if("Captain")
+			prepared_job = pick(/obj/item/restraints/handcuffs, /obj/item/gun/ballistic/automatic/pistol/deagle/gold, /obj/item/gun/energy/e_gun/stun)
```

Comment:
I'm fairly certain that /obj/item/gun/energy/e_gun/stun is the ERT rifle that still has electrodes.